label_studio_ml/examples/tesseract/README.md

```diff
@@ -1,30 +1,50 @@
-## Interactive BBOX OCR using Tesseract
-Using an OCR engine for Interactive ML-Assisted Labelling, this functionality
-can speed up annotation for layout detection, classification and recognition
+<!--
+---
+title: Interactive bounding boxes OCR with Tesseract 
+type: blog
+tier: all
+order: 55
+meta_title: Interactive bounding boxes OCR in Label Studio with a Tesseract backend
+meta_description: Tutorial for how to use Label Studio and Tesseract to assist with your OCR projects
+categories:
+    - tutorial
```

Comment:
do we really need for "tutorial" category?

Reply:
Removed tutorial from category lists